src/App.js

```diff
@@ -1,5 +1,24 @@
-function App() {
-  return <div className="App"></div>;
+import { useState } from 'react';
+import styled from 'styled-components';
+import Wishlist from './Wishlist.js';
+import AddListItem from './AddListItem.js';
+
+export default function App() {
+  const [listItems, setListItems] = useState([]);
+
+  return (
+    <>
+      <Heading>Diving Wishlist</Heading>
+      <AddListItem onAddItem={AddItem} />
+      <Wishlist listItems={listItems} />
+    </>
+  );
+
+  function AddItem(destination, notes) {
+    setListItems([{ destination: destination, notes: notes }, ...listItems]);
```

Comment:
It does not really make a difference, but you are adding each item to the beginning of the array. As long as you are not sorting your items, the latest one is rendered first.

Reply:
Yes, I actually add the items to the beginning of the array on purpose, so that the newest item always appears at the top of the list. :)